backend/src/main/java/saci/application/model/Answer.java

```diff
@@ -0,0 +1,22 @@
+package saci.application.model;
+
+public class Answer {
+    private Integer knowledgeId;
+    private Integer answer;
+
+    public Integer getKnowledgeId() {
+        return knowledgeId;
+    }
+
+    public void setKnowledgeId(Integer knowledgeId) {
```

Comment:
As Vitor said you can erase this gets and sets and use @Data annotation which automatically creates them.
If you only need gets you can use @Getter and if you only use sets use @Setter.

Reply:
Please do this for all the classes with the same manual get/set.